scripts/snapshot-restore-balance/filter_account.json

```diff
@@ -0,0 +1,14 @@
+[
```

Comment:
I don't think we want to filter any account for now, considering we have re-designed the token allocation